### ISSUE TITLE
segnext复现报凑

### ISSUE THREAD

**Issue by djl00** [PaddlePaddle/PaddleSeg#3769]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

我在cityscapes上进行segnext模型复现时，运行报错ValueError: operands could not be broadcast together with shapes (1024,1024) (3,) (1024,1024)
，配置文件是这样的：希望能得到解答，谢谢！
_base_: '../_base_/cityscapes.yml'

batch_size: 2
iters: 160000

train_dataset:
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.5
      max_scale_factor: 2.0
      scale_step_size: 0.25
    - type: RandomPaddingCrop
      crop_size: [1024, 1024]
      category_max_ratio: 0.75
    - type: RandomHorizontalFlip
    - type: RandomDistort
    - type: Normalize
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]

val_dataset:
  transforms:
    - type: Normalize
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]

optimizer:
  _inherited_: False
  type: AdamW
  weight_decay: 0.01

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

lr_scheduler:
  type: PolynomialDecay
  warmup_iters: 1500
  warmup_start_lr: 1.0e-6
  learning_rate: 0.0006
  end_lr: 0.0
  power: 1.0

model:
  type: SegNeXt
  backbone:
    type: MSCAN_T
    pretrained: https://paddleseg.bj.bcebos.com/dygraph/backbone/mscan_t_imagenet_pretrained.zip
  num_classes: 19
  decoder_cfg:
    channels: 256
    ham_channels: 256
    ham_kwargs:
      MD_R: 16
    dropout_rate: 0.1
    align_corners: False
